spec/rails_helper.rb

```diff
@@ -95,6 +110,35 @@ def get(path, headers: nil, sign_with: nil, **args)
     stub_jsonld_contexts!
   end
 
+  config.before :suite do
+    if RUN_SYSTEM_SPECS
```

Comment:
I think instead of the env var conditional here, you could do something like `config.before :suite, type: :system`, which we could eventually migrate to a tag if need.

Reply:
Unfortunately this does not work, `before :suite` does not support tag filtering. It kind of makes sense as this is launched before the whole suite, not before a specific spec or group.